doc/quick-start-user-guide.md

```diff
@@ -68,30 +60,65 @@ clientaddress = "192.168.56.101:24007"
 etcdcurls = "http://192.168.56.101:2379"
 etcdpurls = "http://192.168.56.101:2380"
 ```
-
 Replace the IP address accordingly on each node.
 
-**Start glusterd2 process:** Glusterd2 is not a daemon and currently can run only in the foreground.
+### Using external etcd instead of embedded etcd.
+
+By default glusterd2 uses embedded etcd, but glusterd2 also supports usage of external etcd.
+
+To provide external etcd details. Edit glusterd2 config file by adding `noembed` option and specifying
+the etcd endpoint:
+
+```toml
+etcdendpoints = "http://[ip_address]:[port]"
+noembed = true
+```
+Provide ip-address and port of node where etcd is running.
+
+### Running glusterd2 for RPM installation
+
+**Enable glusterd2 service:** Enable glusterd2 service to ensure that glusterd2 service starts automatically when system starts.
```

Comment:
There's no meaning to `enable glusterd2 service`, it's just that systemctl operation is named so.

`To ensure..., run:`